### PR TITLE
feat: Add product reporting

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: src/script/view_model/CallingViewModel.ts [hunk 1]
```diff
@@ -138,6 +141,7 @@ export class CallingViewModel {
         }
         ring(call);
       });
+      amplify.publish(WebAppEvents.ANALYTICS.EVENT, EventName.CALLING.INITIATED_CALL, {callType});
```
Comment (lipis) [inline src/script/view_model/CallingViewModel.ts]: @bennyn That's the new event.

File: src/script/tracking/EventTrackingRepository.ts [hunk 2]
```diff
@@ -78,7 +78,7 @@ export class EventTrackingRepository {
   }
 
   async init(privacyPreference: boolean): Promise<void> {
-    this.privacyPreference = privacyPreference;
+    this.privacyPreference = privacyPreference || this.userRepository.isTeam();
```
Comment (lipis) [inline src/script/tracking/EventTrackingRepository.ts:73]: That's the check @bennyn

File: src/script/tracking/Segmentations.ts [hunk 3]
```diff
@@ -20,7 +20,7 @@
 /**
  * Enumeration of properties that should get send with every user tracking call.
  */
-export const SuperProperty = {
+export const Segmantations = {
```
Comment (bennycode) [inline src/script/tracking/Segmentations.ts]: Use Signular here.

File: src/script/telemetry/app_init/AppInitTelemetry.ts [hunk 4]
```diff
@@ -40,39 +39,33 @@ export class AppInitTelemetry {
...
+  addStatistic(statistic: AppInitStatisticsValue, value: string | number, bucket_size: number): void {
     this.statistics.add(statistic, value, bucket_size);
   }
 
-  get_statistics(): AppStatistics {
+  getStatistics(): AppStatistics {
     return this.statistics.get();
   }
 
-  get_timings(): AppTimings {
```
Comment (lipis) [inline src/script/telemetry/app_init/AppInitTelemetry.ts:51]: wasn't used anywhere.. if needed we can add it back later.

**Review by bennyn — APPROVED** [review]

**Pull request merged**